File: 프로그래머스/Lv.1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cs
public class Solution {
    public string solution(string[] seoul) {
        string answer = "";
        for(int i =0; i < seoul.Length ; i++)
        {
            if(seoul[i] == "Kim")
            {
                answer = string.Format("김서방은 {0}에 있다",i);
                break;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String answer = "";
        String compareName = "Kim";
        for(int i = 0; i < seoul.length; i++)
        {
            if(seoul[i].equals(compareName))
            {
                
                answer = "김서방은 " + i + "에 있다";
                break;
            }
        }
        return answer;
    }
}